pdm.lock

```diff
@@ -1144,88 +1144,88 @@ content_hash = "sha256:b0f4d25f97b612604c8f853c414dc17ac281a6b4bbd14972bdb8e945a
-    {url = "https://files.pythonhosted.org/packages/e4/47/22c0a0ce8194d5a6821d5d5360ed8279b52b62a28a877a26fb28d95786b9/pydantic_core-0.28.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:4919471d4f21e1c6ad11a205a614ac622d93e23d8dce94f70832b43305fcb89a"},
-    {url = "https://files.pythonhosted.org/packages/e8/24/91999d97c83f9a01b8ae81c59d76d72c037d3c3c92565b7143680f93727d/pydantic_core-0.28.1-cp39-cp39-musllinux_1_1_aarch64.whl", hash = "sha256:2a4955b9b581fcf2bd5a6ae042e73f63cdeae9f06c3abe0d4be17989147bcd25"},
-    {url = "https://files.pythonhosted.org/packages/e9/24/e25fa1f41440ccbc40fb365866de767993e668095f4c6be6eccde6652fe5/pydantic_core-0.28.1-cp310-none-win32.whl", hash = "sha256:6fb6f7e16bb50db0688a7f76a8d640d7b4051719ba1683042f6f24dcc1b20fb1"},
-    {url = "https://files.pythonhosted.org/packages/ea/5b/3c5adf42b1816d7a5249af1699d9ea80302b9437679d91748f82fbdc9a89/pydantic_core-0.28.1-pp38-pypy38_pp73-macosx_10_7_x86_64.whl", hash = "sha256:6a5abaf2eaf89031cb288073502f092fc564a6d625603009e0e228d47c7dc138"},
-    {url = "https://files.pythonhosted.org/packages/ea/8f/e03570cedd676f259bd8a09d528ff5b691e2d002bdd57db67a5cbbd1f6b4/pydantic_core-0.28.1-cp310-cp310-manylinux_2_24_ppc64le.whl", hash = "sha256:fe19282afc3a93075a4430825143bb97ddfd133074291ceac24732c8a73ef664"},
-    {url = "https://files.pythonhosted.org/packages/f6/3a/cb42b628c17e7a103ad5e47788c25833a18d45db2388a967a898c1639510/pydantic_core-0.28.1-pp38-pypy38_pp73-win_amd64.whl", hash = "sha256:d800854ee6e92671c6a79d992796baf32974e35060baff42e92dc42c926b0b36"},
-    {url = "https://files.pythonhosted.org/packages/fd/dd/36a65fa2802f932ede3217f128461de2f639935621306c686e87e872eab4/pydantic_core-0.28.1-cp38-cp38-macosx_10_7_x86_64.whl", hash = "sha256:03df812b5fbe22fc1e98a4a6a156d8dc4a2cf8c126e792d2f68be64b1cb545d0"},
-    {url = "https://files.pythonhosted.org/packages/ff/01/c632172ae41b889bcdafe9e7258507d76bb0fa86cfd92011eece069df2ca/pydantic_core-0.28.1-cp311-cp311-macosx_11_0_arm64.whl", hash = "sha256:b469df56352af3c8409977fed98e6faad2b81b157fd1fe7daec89a6aa7b9e172"},
-    {url = "https://files.pythonhosted.org/packages/ff/14/da5d6cecf2cb6ad5af36d5a7593b87d6415ed1412a9f49116e8c2a760ae4/pydantic_core-0.28.1-cp310-cp310-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:80433de8f4334c3acbfaacf6de8cd2c6f57d71068a7418b2be4e08465b6c1e37"},
-    {url = "https://files.pythonhosted.org/packages/ff/7d/2310ebe71aaedbefbe00886107a140fa9103e10f8fa6d5ebdb413bdd75d4/pydantic_core-0.28.1-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:82b54ec2ab627ddb78aa9b8c23d0a525308c95b68d4316145dc925751424c346"},
+"pydantic-core 0.29.0" = [
+    {url = "https://files.pythonhosted.org/packages/02/49/823072252731176db03873ab4679b2e80d7cceb5c9e3e0b6ec723f31c8bd/pydantic_core-0.29.0-cp311-cp311-musllinux_1_1_x86_64.whl", hash = "sha256:5b4ca92cac896c8e13b12e42109304ae385e48dde4b321d04238ba56997063c0"},
```

Comment:
looks like you refreshed the whole file, better in general to do `pdm add pydantic-core==0.29.0` and it won't update everything.

Reply:
that's what I did, I think there's just a lot of pydantic core lines

Reply:
One line per wheel

Reply:
🤦 🙏